pkg/operator/configobservation/auth/auth_serviceaccountissuer.go

```diff
@@ -71,6 +71,12 @@ func observedConfig(existingConfig map[string]interface{},
 		existingActiveIssuer = existingIssuers[0]
 	}
 
+	// if the issuer is not set, it is safe to assume it is being defaulted by the config override and we don't have
+	// to trigger revision rollout just to sync operator status with config.
+	if len(existingActiveIssuer) == 0 {
```

Comment:
This is needed because the config this observer sync on is a lie :-) There is never an empty existingActiveIssuer; if it is, it has defaulted.
This change will default it here, so this observer won't trigger a new revision, thinking there is now a "new" value for the issuer.